crates/re_viewer/src/misc/item.rs

```diff
@@ -38,7 +38,7 @@ impl Item {
             Item::InstancePath(space_view_id, _) => space_view_id
                 .map(|space_view_id| blueprint.viewport.space_view(&space_view_id).is_some())
                 .unwrap_or(true),
-            Item::RowId(row_id) => log_db.get_log_msg(row_id).is_some(),
+            Item::RowId(row_id) => log_db.get_entity_op_msg(row_id).is_some(),
```

Comment:
can do better here

Reply:
I checked, and `Item::RowId` is unused. I pushed a commit to remove it.